xdp-dump/xdpdump.c

```diff
@@ -416,27 +416,119 @@ static bool capture_on_legacy_interface(struct dumpopt *cfg)
 	return rc;
 }
 
+static size_t find_func_matches(const struct btf *btf,
```

Comment:
Missing function marker

```
/*****************************************************************************
 * find_func_matches()
 *****************************************************************************/
```

Reply:
Just out of curiosity, is there a particular reason you add these headings? I.e., does your editor do something specific with them, or is it purely aesthetics?

Reply:
It's my OCD kicking in, can't stand long files without a clear function/data boundary.

Thanks for making all the review changes, I guess it kept you buys based on the number of commits :)

Reply:
Right fair enough. And yeah, this has kept me busy for a bit, but getting close now! Thanks for reviewing :)